src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1023,11 +1023,21 @@ public String getKeyspaceName()
         return keyspace.getName();
     }
 
+    public Tracker getData()
```

Comment:
Consider calling this `getSSTableTracker` and reducing its visibility.

Reply:
Done.

Reply:
Actually, don't we already have a `getTracker` method?